ci-operator/step-registry/assisted/nutanix/postinstall/assisted-nutanix-postinstall-commands.sh

```diff
@@ -51,10 +51,209 @@ stringData:
 EOF
 echo "machine API credentials created"
 
-until \
-  oc wait --all=true clusteroperator --for='condition=Available=True' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Progressing=False' >/dev/null && \
-  oc wait --all=true clusteroperator --for='condition=Degraded=False' >/dev/null;  do
-    echo "$(date --rfc-3339=seconds) Clusteroperators not yet ready"
-    sleep 1s
+version=$(oc version | grep -oE 'Server Version: ([0-9]+\.[0-9]+)' | sed 's/Server Version: //')
+resource_timeout_seconds=300
+
+# Do the following if OCP version is >=4.13
```

Comment:
why we need this configmap only for 4.13?

Reply:
I'll add it to the comment